app/src/main/java/fr/free/nrw/commons/CommonsApplication.java

```diff
@@ -54,6 +54,14 @@
     @Inject @Named("application_preferences") SharedPreferences applicationPrefs;
     @Inject @Named("prefs") SharedPreferences otherPrefs;
 
+    /**
+     * Constants begin
+     */
+    public class REQUEST_CODES {
```

Comment:
Umm, is there a need for an inner class? We can just put the constant in the base class, right?

Reply:
I had in mind that we should put all constants belonging to a particular something, together, so that when they are large in number it will be easy to reference in the classes using them. Anyways I'l put it in the base class for now. Later on we can discuss and extract out if needed :-)